fe/fe-core/src/main/java/org/apache/doris/common/proc/TabletsProcDir.java

```diff
@@ -45,7 +45,7 @@ public class TabletsProcDir implements ProcDirInterface {
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("TabletId").add("ReplicaId").add("BackendId").add("SchemaHash").add("Version")
             .add("LstSuccessVersion").add("LstFailedVersion").add("LstFailedTime")
-            .add("DataSize").add("RowCount").add("State")
+            .add("LocalDataSize").add("RemoteDataSize").add("RowCount").add("State")
```

Comment:
This would cause compatibility problems, would using DataSize also work?

Reply:
Of course we can use DataSize here, but I think it may be a little confusing when user find DataSize is 0.